metadata.json

```diff
@@ -8,7 +8,10 @@
   "project_page": "https://github.com/rnelson0/puppet-certs",
   "issues_url": "https://github.com/rnelson0/puppet-certs/issues",
   "dependencies": [
-
+    {
```

Comment:
I think I'd prefer to have this a soft dependency and documented in the readme. Something like

```
To use the vault options, you must have a module that is API compatible with <link to forge for vault_lookup> installed. If you are not using vault, this dependency is optional.
```

Reply:
This has been incorporated